internal/controllers/requirements/docker/docker.go

```diff
@@ -69,22 +65,15 @@ func validateIfDockerIsSupported(version string) error {
 }
 
 func execDockerVersion() (string, error) {
```

Comment:
This function should be renamed to something like `getDockerVersion() (string, error)`